src/window.rs

```diff
@@ -333,6 +335,23 @@ impl WindowBuilder {
         self
     }
 
+    /// Build window with resize increment hint. Only implemented on X11 and macOS.
```

Comment:
Nit: Use `## Platform-specific` section

Reply:
Bump.

Reply:
I referenced the setter instead, like every other `with_` does